charts/azuremonitor-containers/values.yaml

```diff
@@ -71,6 +71,10 @@ omsagent:
     ## Applicable for only Azure Stack Edge K8s since it has custom mount path for container logs which will have symlink to /var/log path
     custommountpath: ""
 
+  arcasettings:
```

Comment:
what is "arca" stands for?

Reply:
"arca" stands for Arc-Autonomous, which is our latest product name.
"Arc Autonomous is a virtual appliance installable on ASZ/HCI or VMWare that acts as the hosting platform for Azure Arc services running on-prem."  -- from [Arc Autonomous Business Model  ](https://microsoft.sharepoint.com/:w:/r/teams/ASZ/_layouts/15/doc2.aspx?sourcedoc=%7B9222E7A3-3F19-43AB-B6EF-2A3792D2479D%7D&file=Arc%20Autonomous%20Business%20Model_Partner%20Version.docx&action=default&mobileredirect=true&DefaultItemOpen=1&share=IQGj5yKSGT-rQ7bvKjeS0kedAcAO5snXzBW6F7LQnRfhiPM)